app/views/interactions/index.html.erb

```diff
@@ -7,6 +7,11 @@
 <div class="page-title">
   <h2><%= @service.label %></h2>
 </div>
+
+<% if flash[:success_action] %>
```

Comment:
I think this duplicates the flash feature in `govuk_admin_template` (https://github.com/alphagov/govuk_admin_template#flash-messages). Or is that intentional?